panels/dev-service/ha-panel-dev-service.html

```diff
@@ -49,25 +50,34 @@
         </app-toolbar>
       </app-header>
 
+      <app-localstorage-document
+        key='panel-dev-service-state-domain'
+        data='{{domain}}'>
+      </app-localstorage-document>
+      <app-localstorage-document
+        key='panel-dev-service-state-service'
+        data='{{service}}'>
+      </app-localstorage-document>
+      <app-localstorage-document
+        key='panel-dev-service-state-servicedata'
+        data='{{serviceData}}'>
+      </app-localstorage-document>
+
       <div class$='[[computeFormClasses(narrow)]]'>
```

Comment:
Means we also no longer have to include the custom style `iron-flex` and `iron-positioning`